frontend/src/components/ElectionForm/ElectionDetails.tsx

```diff
@@ -3,25 +3,23 @@ import { useState } from "react"
 import Grid from "@mui/material/Grid";
 import TextField from "@mui/material/TextField";
 import FormControl from "@mui/material/FormControl";
-import { FormHelperText, InputLabel } from "@mui/material"
+import { FormHelperText, InputLabel, MenuItem, Select } from "@mui/material"
 import { StyledButton } from '../styles';
 import { Input } from '@mui/material';
-
+import { DateTime } from 'luxon'
+import { timeZones } from './TimeZones'
+import { useLocalStorage } from '../../hooks/useLocalStorage';
 export default function ElectionDetails({ election, applyElectionUpdate, getStyle, setPageNumber }) {
-    const dateAsInputString = (date) => {
-        if (date == null) return ''
-        date = new Date(date)
-        // Remove time zone offset before switching to ISO
-        var s = (new Date(date.getTime() - date.getTimezoneOffset() * 60000).toISOString());
-        s = s.replace(':00.000Z', '')
-        return s
-    }
+
     const [errors, setErrors] = useState({
         title: '',
         description: '',
         startTime: '',
         endTime: '',
     })
+
+    const [timeZone, setTimeZone] = useLocalStorage('timezone',DateTime.now().zone.name)
```

Comment:
Interesting that we're using localStorage here. So every time they create an election it'll remember what timezone they used, and then that'll be the default for the next election?

I think always using the device timezone would be a better default since a non-local timezone is likely to be a one-off, but I'm conflicted on that one

Reply:
Hmm, so I had just used state here originally but ran in the issue where if you go to the next page and then back it would reset the time zone to the device's which was annoying. Maybe the least annoying thing would be to add it to a parameter in the settings object and default it to your local browser. Then it would persist just for that election across all users/devices.

Reply:
Yeah I like that, I could see that information being handy in the future